README.md

```diff
@@ -3762,3 +4352,412 @@ return Http::withOptions([
 
 Tip given by [@raditzfarhan](https://github.com/raditzfarhan)
 
+### Test that doesn't assert anything 
+Test that doesn't assert anything, just launch something which may or may not throw an exception
+
+```php
+class MigrationsTest extends TestCase
+{
+    public function test_successful_foreign_key_in_migrations()
+    {
+        // We just test if the migrations succeeds or thwors an exception
```

Comment:
@krekas throws* (typo)

Also, there should be a line of some action after `expectNotToPerformAssertions()` so readers would understand the possible use-case.